fedimint-eventlog/src/lib.rs

```diff
@@ -440,6 +441,101 @@ where
     }
 }
 
+/// Filters the given `PersistedLogEntry` slice by the `EventKind` and
+/// `ModuleKind`.
+pub fn filter_events<'a, I>(
+    all_events: I,
+    event_kind: EventKind,
```

Comment:
`s/filter_events/`filter_events_by_kind`, and then `module_kind` should probably be before `event_kind`, as it has a higher scope.